spec/unit/puppet/provider/vrf/cisco_ios_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+module Puppet::Provider::Vrf; end
+require 'puppet/provider/vrf/cisco_ios'
+
+RSpec.describe Puppet::Provider::Vrf::CiscoIos do
+  def self.load_test_data
+    PuppetX::CiscoIOS::Utility.load_yaml(File.expand_path(__dir__) + '/test_data.yaml', false)
+  end
+
+  it_behaves_like 'resources parsed from cli'
+
+  context 'Create/Delete tests:' do
+    load_test_data['default']['create_tests'].each do |test_name, test|
+      it test_name.to_s do
+        fake_device(test['device'], test['family'])
+        if test['commands'].size.zero?
+          expect { described_class.create_commands_from_instance(test['instance']) }.to raise_error(%r{.*})
+        else
+          result = []
+          described_class.create_commands_from_instance(test['instance']).each { |x| result << x.squeeze(' ') }
+          expect(result).to eq test['commands']
+        end
+      end
+    end
+  end
+
+  context 'Create/Delete tests:' do
```

Comment:
This should be `Update tests:` or something similar to differentiate it from the test's above.